[IAM-5434] extend section for sso cert rotation (#2103)
saucelabs/sauce-docs: docs/basics/sso/setting-up-sso.md

Sauce Labs Service Provider does not require any extra attributes in the SAML Response.

### Identity Provider Certificate Rotation

SAML certificates have a predetermined expiration date. It is possible that your identity provider will notify you in advance or provide an alert regarding the certificate's expiry.

> If the certificate for the Sauce Labs app from your identity provider is nearing expiration or has already expired, please follow these steps:

Author: fixed

1. Generate a new certificate for the Sauce Labs app within your identity provider. This new certificate will replace the expired one and will be reflected in the identity provider metadata.

> 2. Export or copy the updated metadata file from your identity provider. Then, proceed to upload it in Sauce Labs by following [these steps](/basics/sso/setting-up-sso/#integrating-with-sauce-labs-service-provider).

Reviewer: ```suggestion
2. Export or copy the updated metadata file from your identity provider, then upload it to Sauce Labs. For information on how to upload the metadata file, see [Integrating With Sauce Labs Service Provider](/basics/sso/setting-up-sso/#integrating-with-sauce-labs-service-provider).
```

### Sauce Labs Certificate Rotation

:::note
This section is relevant only for identity providers that enabled encryption of SAML Assertions.
:::

If you enabled encryption of SAML Assertions in your IdP, you must upload the certificate that is provided in Sauce Labs [metadata](https://accounts.saucelabs.com/am/sso/metadata/https%3A%2F%2Faccounts.saucelabs.com%2Fsp).

The certificate is valid for one year. **120 days** before the expiration date the new certificate will be available in Sauce Labs [metadata](https://accounts.saucelabs.com/am/sso/metadata/https%3A%2F%2Faccounts.saucelabs.com%2Fsp).

> During the rotation period the Sauce Labs metadata contain two valid certificates. If you are unsure which certificate to upload, copy the content of the `<ds:X509Certificate>` tag and paste it into [one of the oneline x.509 certificate decoders](https://www.sslshopper.com/certificate-decoder.html). Then, choose the certificate that has a longer validity period.

Reviewer: ```suggestion
During the rotation period, the Sauce Labs metadata contains two valid certificates. If you are unsure which certificate to upload, copy the content of the `<ds:X509Certificate>` tag and paste it into a [certificate decoder](https://www.sslshopper.com/certificate-decoder.html). Then, select the certificate that has a longer validity period.
```

> To convert the certificate from the metadata into the PEM format required by most identity providers, you can utilize the [online tool provided by OneLogin](https://www.samltool.com/format_x509cert.php). Follow these steps:

Reviewer: ```suggestion
To convert the certificate from the metadata into the PEM format required by most identity providers, you can use the [online tool provided by OneLogin](https://www.samltool.com/format_x509cert.php). Follow these steps:
```

> 1. Copy the content of the `<ds:X509Certificate>` tag and paste it into the **X.509 cert** text box. Click **FORMAT X.509 CERTIFICATE**.

Reviewer: ```suggestion
1. Copy the content of the `<ds:X509Certificate>` tag, paste it into the **X.509 cert** text box, then click **FORMAT X.509 CERTIFICATE**.
```

> 2. Copy the content of the **X.509 cert with header** textbox and save it in a text file, e.g. `saucelabs-sso.pem`.

Author: fixed

> 3. Finally, upload the new certificate in your IdP.

Reviewer: ```suggestion
3. Upload the new certificate in your IdP.
```
